content/md/first-steps/manage-families.md

```diff
@@ -183,12 +181,39 @@ Moreover, the attribute cannot be scopable, localizable and locale specific.
 The maximum number of variant axes for each variant level is less than **5**.
 :::
 
+Then, a screen to distribute the attributes between the different levels (`common attributes`, `variant attributes level 1` and `variant attributes level 2`) for the family variant opens
+1.  Drag and drop attributes or attribute groups from the `common attributes` to the selected variant level to have these attributes managed at each variant level
+![Display a family variant](../img/Settings_Families_Variant_DistributeAttributes.png)
+1.  Click on `Save`, the family variant is updated
+
+::: info
+By default, the `common attributes` are all the attributes of the family, except the variant axes attributes (available for each variant level) and the attributes with the property `unique value` at `Yes` which are at the last variant level = product level (for example the `SKU` and `EAN`).
+:::
+
+
 ## Edit a family variant
 
+1.  In `Settings/Families`, choose a family and open the `Variants` tab
+1.  Click on a family variant
+1.  The family variant edit page is displayed
+![Display a family variant](../img/Settings_Families_Variant_Display.png)
+1.  You can move attributes or attribute groups by a drag and drop from the `common attributes` to the selected variant level or from the `variant attributes level 1` to the `variant attributes level 2`
+1.  You can remove an attribute from a variant level by clicking on the cross
+1.  A confirmation opens
+![Display a family variant](../img/Settings_Families_Variant_EditRemove.png)
+1.  Click on `OK` to confirm the remove
+1.  When you have made all your updates on the family variant attributes, click on `Save`
+![Display a family variant](../img/Settings_Families_Variant_Edit.png)
+
 ::: info
-This feature will be available soon. For the moment, you can only update the family variant by a dedicated import `Family variant import` available in CSV and XLSX.
+Move attributes from `common attributes` to the selected variant level or from the `variant attributes level 1` to the `variant attributes level 2` mean than these attributes will be now managed at a level under (not common but variant), the values for these attributes are kept and moved. For the `Clothing` example, if you move the attribute `Description` from the `common attributes` to the `variant attributes level 1`, it means than now the description can be managed by color.
```

Comment:
**Moving** .... **means** **that** these attributes will now **be** managed
it means **that** now the description
(in bold, the words I changes or moved)

Reply:
ah bah oui that :)